kratos/tests/test_levelset_convection.py

```diff
@@ -175,23 +169,18 @@ def test_levelset_convection_BFECC_algebraic(self):
         linear_solver = linear_solver_factory.ConstructSolver(
             KratosMultiphysics.Parameters("""{"solver_type" : "skyline_lu_factorization"}"""))
 
-        model_part.CloneTimeStep(1.0)
+        model_part.CloneTimeStep(10.0)
 
         KratosMultiphysics.FindGlobalNodalNeighboursProcess(model_part).Execute()
 
-        KratosMultiphysics.ComputeNonHistoricalNodalGradientProcess(
-            model_part,
-            KratosMultiphysics.DISTANCE,
-            KratosMultiphysics.DISTANCE_GRADIENT,
-            KratosMultiphysics.NODAL_AREA).Execute()
-
         levelset_convection_settings = KratosMultiphysics.Parameters("""{
-            "max_CFL" : 1.0,
+            "max_CFL" : 0.2,

```

Comment:
Shouldn't this work with CFL 1.0?

Reply:
Including the anti-diffusive terms, the good practice is to keep CFL below 1.0 if one is going to use an explicit approach (forward Euler). Here, the anti-diffusivity term is treated explicitly (though we use TIME_INTEGRATION_THETA=0.5 by default).